### DISCUSSION TITLE
Timeseries collections

### DISCUSSION THREAD

**Discussion by Torgen** [meteor/meteor#11937] [Feature Requests]
Meteor 5 supports timeseries collections, which store data more efficiently when the metadata fields repeat: 
https://docs.mongodb.com/manual/core/timeseries-collections/ 
You make a collection a timeseries collection by passing certain options to the createCollection call. Meteor doesn't give us direct access to the arguments to the createCollection call. Can we get an option to the Mongo.Collection constructor to make the collection a timeseries collection? If the collection already exists but isn't a timeseries collection, I expect it to fail in the same way that createIndex fails if you already have an index on the given columns with conflicting options.

**Comment by radekmie** [comment]
I haven't tried it with Meteor yet, but I'm not certain it'll work. Timeseries collections have [some limitations](https://docs.mongodb.com/manual/core/timeseries/timeseries-limitations/) and while oplog is not listed there, it may not work at all. Have you tried it maybe?

**Reply by Torgen** [reply]
I tried manually creating a timeseries collection and subscribing to it, and while the initial load worked, it didn't generate live updates. The docs do mention a timeseries collection being essentially a view around internal collections, so I guess the oplog entries appear as belonging to the internal collections and not to the timeseries collection.
